### DISCUSSION TITLE
Could not find files for / in .next/build-manifest.json

### DISCUSSION THREAD

**Discussion by omar-dulaimi** [vercel/next.js#18368] [Help]
Hello,
I tried to` NODE_ENV=production npm run build` and `NODE_ENV=production npm start`
but with no luck to locating the actually masked issue here.

Any help?

**Comment by omar-dulaimi** [comment]
I tried to upgrade to Next10 but that didn't uncover any silent errors.

**Comment by Manoj-nathwani** [comment]
Heads up - I've managed to fix this by updating webpack via
```
yarn add webpack --dev
```
